__tests__/app.test.js

```diff
@@ -85,21 +89,70 @@ describe("core tasks", () => {
     });
     // error handling tests
     it("GET: responds with 404 and when id not present", () => {
-        return request(app)
-        .get('/api/articles/99999')
+      return request(app)
+        .get("/api/articles/99999")
         .expect(404)
-        .then(({body}) => {
-            const error = body.message
-            expect(error).toBe('Article not found')
-        })
+        .then(({ body }) => {
+          const error = body.message;
+          expect(error).toBe("Article not found");
+        });
     });
     it("GET: responds with 400 and when id not number", () => {
       return request(app)
         .get("/api/articles/banana")
         .expect(400)
         .then(({ body }) => {
-          const error = body.message
-          expect(error).toBe('Bad request')
+          const error = body.message;
+          expect(error).toBe("Bad request");
+        });
+    });
+  });
+
+  describe("task 6", () => {
```

Comment:
It may be worth swapping this for the endpoint URL (just as its a portfolio piece that a future employer may wish to look at)